src/Builder.php

```diff
@@ -564,7 +564,7 @@ protected function getTotalCount($results)
     public function applyAfterRawSearchCallback($results)
     {
         if ($this->afterRawSearchCallback) {
-            call_user_func($this->afterRawSearchCallback, $results);
+            $results = call_user_func($this->afterRawSearchCallback, $results);
```

Comment:
@The-Rabak If the afterRawSearchCallback does not return anything, then it should still use the original $result value (similar to the afterQuery-method on eloquent's query builder). Otherwise, this will be a breaking change.
```suggestion
            $results =  call_user_func($this->afterRawSearchCallback, $results) ?: $results;
```